portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -50,11 +53,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PreparedQuery results = datastore.prepare(query);
     FetchOptions fetchOptions = FetchOptions.Builder.withLimit(maxComments);
 
-    ArrayList<String> comments = new ArrayList<>();
+    ArrayList<HashMap> comments = new ArrayList<>();
```

Comment:
This is also changing the structure of the JSON so you also need to change your JS to handle this.

Reply:
I see. Good thinking on making PRs small but merging this change would make the merged branch into a broken state (if you were to deploy your app, something would crash). It does make sense to have it as part of separate commit but in the same PR.